test/test_sim_utils.py

```diff
@@ -21,26 +23,55 @@
     not built_with_bullet,
     reason="ArticulatedObject API requires Bullet physics.",
 )
+@pytest.mark.skipif(
+    not osp.exists("data/test_assets/scenes/plane.glb"),
+    reason="Requires the plane.glb habitat test asset",
+)
 @pytest.mark.parametrize(
     "support_margin",
     [0.0, 0.04, 0.1],
 )
 @pytest.mark.parametrize("obj_margin", [0.0, 0.04, 0.1])
-def test_snap_down(support_margin, obj_margin):
+@pytest.mark.parametrize("stage_support", [True, False])
+def test_snap_down(support_margin, obj_margin, stage_support):
     """
     Test snapping objects onto stages and other assets.
     """
 
     mm = MetadataMediator()
 
     otm = mm.object_template_manager
+    stm = mm.stage_template_manager
     # setup a cube ground plane object config
     cube_template_handle = otm.get_template_handles("cubeSolid")[0]
     cube_stage_template_handle = "cube_stage_object"
-    cube_template = otm.get_template_by_handle(cube_template_handle)
-    cube_template.scale = mn.Vector3(10, 0.05, 10)
-    cube_template.margin = support_margin
-    otm.register_template(cube_template, cube_stage_template_handle)
+    plane_stage_template_handle = "plane_stage"
```

Comment:
Maybe have a helper method called `register_cube_or_plane_template` ? 
I am not sure what this added code does, maybe a comment would help ?